Ginger/GingerCoreTest/Misc/DataBaseTest.cs

```diff
@@ -362,5 +362,53 @@ public void RenameDBName()
             Assert.AreEqual(actDB.DBName, db.Name, "Names");
 
         }
+
+        [TestMethod]
+        [Timeout(60000)]
+        public void TestHidePasswordFromString()
+        {
+            //Access Db
+            string ConString = "Provider=Microsoft.ACE.OLEDB.12.0;Data Source=C:/myAccessFile.accdb;Jet OLEDB:Database Password=MyDbPassword;";
```

Comment:
correct way of writing unit test should include the 3 A's, //Arrange, //Act and //Assert:
//Arrange
here you should place the preperation code, like:
string ConString = "Provider=Microsoft.ACE.OLEDB.12.0;Data Source=C:/myAccessFile.accdb;Jet OLEDB:Database Password=MyDbPassword;";

//Act
here you place the code you testing, like:
string Result = General.HidePasswordFromString(ConString);

//Assert
here, you add all asserts, like:
Assert.AreNotEqual(ConString, Result);
Assert.AreEqual(Result.Contains("Password=*****"), true);

please re desing the test with the 3 A's, you also can/should split it to unit test per connection string type